Documentation/operations/troubleshooting.rst

```diff
@@ -713,6 +713,38 @@ When running in :ref:`arch_direct_routing` mode:
 
 4. Verify that the firewall on each node permits to route the endpoint IPs.
 
+Traffic to Pods of a Service is unbalanced
+------------------------------------------
+
+Symptom
+~~~~~~~
+
+Endpoint to endpoint communication between nodes is successful, but connections
+and / or load appears unbalanced to one or more Pods that make up a Service.
+One or more Pods receives more traffic than the others.
+
+Troubleshooting steps:
+~~~~~~~~~~~~~~~~~~~~~~
+
+Confirm the load balancing mode enabled for Services in the cluster, set by the
+``hostServices.enabled`` Helm flag or the ``enable-host-reachable-services``
+flag passed to the Cilium agent. More information found on Host-Reachable
+services in :ref:`host-services`. Another option is to consider whether running
```

Comment:
I think that `host-services` should be superseded by the `kubeproxy-free` page, so I'd refrain from linking to it.